examples/config/laptop_config.yml

```diff
@@ -190,3 +190,6 @@ TXTwoPointFourier:
 TXSimpleMask:
     depth_cut : 23.5
     bright_object_max: 10.0
+
+TXMapCorrelations:
+    supreme_path_root: data/example/inputs/supreme
```

Comment:
can we add "outlier fraction" and "nbin" here just to make it clear you can set it?

Reply:
done.